### DISCUSSION TITLE
articles/dalifornia

### DISCUSSION THREAD

**Discussion by giscus** [hutusi/hutusi.github.com#188] [Comments]
# articles/dalifornia

年初闲来无事，看别人的总结。其中DIYGod的总结让我印象很深刻，他每年给自己制定OKR，其中KR部分很详细，且Smart化，这种将自己的生活量化规划的方式是个不错的实践，因此我也尝试着按照健康、家庭、成长、工作几个方面来规划OKR，在家庭Objective一项分解的KR里，列上“今年安排家庭旅游至少3次”，因此也想利用下春节假期来实现第一个KR，跟妻子商量目的地，在泰国、哈尔滨等地纠结了几天后，终于在临春节前几天定下去云南。

https://hutusi.com/articles/dalifornia

**Comment by philip-fhjkladsh** [comment]
这照片拍的真不错。

**Reply by hutusi** [reply]
谢谢：）
